dotCMS/src/main/java/com/dotmarketing/common/reindex/BulkProcessorListener.java

```diff
@@ -66,6 +65,7 @@ public void beforeBulk(final long executionId, final BulkRequest request) {
 
     @Override
     public void afterBulk(final long executionId, final BulkRequest request, final BulkResponse response) {
+        Logger.debug(this.getClass(), "Bulk process completed");
```

Comment:
we used to use a supplier: Logger.debug(this.getClass(), ()->"Bulk process completed");